src/lake/Lake/Build/Common.lean

```diff
@@ -118,28 +118,21 @@ be `weakArgs` to avoid build artifact incompatibility between systems
 You can add more components to the trace via `extraDepTrace`,
 which will be computed in the resulting `BuildJob` before building.
 -/
-@[inline] def buildCToO (name : String)
+@[inline] def buildO (name : String)
 (oFile : FilePath) (srcJob : BuildJob FilePath)
 (weakArgs traceArgs : Array String := #[]) (compiler : FilePath := "cc")
 (extraDepTrace : BuildM _ := pure BuildTrace.nil) : SchedulerM (BuildJob FilePath) :=
   let extraDepTrace := return mixTrace (← extraDepTrace) (← computeHash traceArgs)
   buildFileAfterDep oFile srcJob (extraDepTrace := extraDepTrace) fun srcFile => do
-    compileCToO name oFile srcFile (weakArgs ++ traceArgs) compiler
+    compileO name oFile srcFile (weakArgs ++ traceArgs) compiler
 
 /-- Build an object file from a source fie job (i.e, a `lean -c` output) using `leanc`. -/
-@[inline] def buildLeanFromCToO (name : String)
+@[inline] def buildLeanO (name : String)
 (oFile : FilePath) (srcJob : BuildJob FilePath)
 (weakArgs traceArgs : Array String := #[]) : SchedulerM (BuildJob FilePath) :=
   let extraDepTrace := return mixTrace (← getLeanTrace) (← computeHash traceArgs)
   buildFileAfterDep oFile srcJob (extraDepTrace := extraDepTrace) fun srcFile => do
-     compileCToO name oFile srcFile (weakArgs ++ traceArgs) (← getLeanc)
-
-@[inline] def buildLeanFromBcToO (name : String)
```

Comment:
I think you may have cut a bit too much from my review. I think you still probably need `buildLeanFromBcToO` to pass the additional arguments and compiler to `compileO`.

Reply:
There are no additional arguments needed for LLVM as I understand it, the -march=native from before is an optimization flag it is not required. And the compiler can just be a normal leanc as long as that leanc points to a clang thing which it does for sure in production and does in dev as well thanks to the CMake change above.